visualizations/plotly/webviz_plotly/__init__.py

```diff
@@ -20,6 +21,21 @@ class Plotly(JSONPageElement):
     """
     def __init__(self, data, layout={}, config={}):
         super(Plotly, self).__init__()
+
```

Comment:
Probably a good idea to add a disclaimer in the docstring of `Plotly`. Something like:
Note: Plotly will override the configuration to remove some modeButtons which are not
useful for the visualizations implemented by webviz. See Plotly.disallowed_buttons for buttons
not included in webviz visualizations.